lib/perl/Genome/Model/Tools/Validation/IdentifyOutliers.pm

```diff
@@ -105,27 +105,38 @@ sub execute {
     my $normal_var_col;
     my $tumor_ref_col;
     my $tumor_var_col;
+    my $linecount = 0;
 
     #scan through file and identify where we think there are outliers
     while(my $line = $fh->getline) {
         chomp $line;
         my @F = split /\t/, $line;
-        
+
         #get fields to query from header
-        if($line =~ /chromosome_name/){
-            $normal_ref_col = List::MoreUtils::first_index {$_ =~ /Normal_ref_count/ } @F;
-            $normal_var_col = List::MoreUtils::first_index {$_ =~ /Normal_var_count/ } @F;
-            $tumor_ref_col = List::MoreUtils::first_index {$_ =~ /Tumor_ref_count/ } @F;
-            $tumor_var_col = List::MoreUtils::first_index {$_ =~ /Tumor_var_count/ } @F;
-
-            print OUTFILE $line . "\tFilterCall\tProb\tLLR\n";
-            if(defined($self->lq_output_file)){
-                print LQFILE $line . "\tFilterCall\tProb\tLLR\n";
+        if($linecount == 0){
```

Comment:
It might be nice to pull this part out to before the loop, since we know it'll happen exactly once.